NetSSL_Win/src/Context.cpp

```diff
@@ -140,13 +140,18 @@ void Context::addTrustedCert(const Poco::Net::X509Certificate& cert)
 }
 
 
-Poco::Net::X509Certificate Context::certificate()
+Poco::Net::X509Certificate Context::certificate(bool mustFindCertificate)
 {
 	if (_pCert)
 		return Poco::Net::X509Certificate(_pCert, true);
 
 	if (_certNameOrPath.empty())
-		throw NoCertificateException("Certificate requested, but no certificate name or path provided");
```

Comment:
I inspected the code and did not find where this exception is logged.

Reply:
Oh sorry, I'm an idiot and wasted your time. Visual Studio logs all exceptions, I just didn't notice that before for whatever reason and thought it has to do with the POCO_IMPLEMENT_EXCEPTION without digging further :-( So my throw in this patch still logs an exception (just without that string). Which obviously improves nothing.  Only way to get no message in VS would be to avoid throwing exceptions completely. Which I guess would need more rework like handling an invalid X509Certificate or something like that. And I'm really not up to providing a patch for that. Don't even know if it would fit in this library or if exceptions are simply used as error codes sometimes.

So can be closed I guess. And thanks for taking a look. At least I learned something.